docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 ---
 
 name: typst-bot
-version: "3.8"
+version: "3.9"
```

Comment:
One last question, is this supposed to be semver? What is this version used for? I'm just thinking that if adding this secrets file is a breaking change, it might need to be 4.0.

Reply:
When I did this, I mistakenly thought that that were the project's version.  I ignored the deprecation warning each time I tested this project.  After some searching, now I have a better idea what it is.  Let me remove this line.

Reply:
OK, I am happy to remove it then.